crud/compare/conditions.lua

```diff
@@ -75,6 +85,11 @@ local funcs_by_symbols = {
     ['<='] = conditions.funcs.le,
     ['>'] = conditions.funcs.gt,
     ['>='] = conditions.funcs.ge,
+    ['<>'] = conditions.funcs.no,
```

Comment:
Does this syntax inspired by some existing database API or not? Not all of these are intuitive.